### DISCUSSION TITLE
Error in deployment after renaming Table Name

### DISCUSSION THREAD

**Discussion by craftanalytix** [TabularEditor/TabularEditor3#1286] [🙋♀️TE3 Q&A]
I have a database in workspace that is working since I first publish. After I updated the metadata by renaming the fact table. I can't deploy anymore and saying "Fact Table does not exist". I fix all relationships, update schema and also calculate refresh and not luck. 

<img width="374" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/3468860/2e8cc23c-20ed-4bbb-bd15-61c1720f761d">

**Comment by mlonsk** [comment]
Hi @RoadDeal 
How are you deploying the model? Are you using Ctrl+S or are you using the deployment wizard in Model>Deployment?

Also when you say you updated the metadata what does that mean? Did you manually edit a file?

**Reply by craftanalytix** [reply]
@mlonsk I have the workspace mode which is my metadata connected to dev workspace. I did all edit while connected to workspace in dev. I have deployed to test after all necessary changes in dev using Model > Deployment. 

Initial deployment the table added doesn't exist yet. Works great with deployment.
Second changes I have renamed the table in a workspace mode with metadata and dev connected. I did deployment again from Dev to Test Model > Deployment. This is the part I am having error.

**Reply by craftanalytix** [reply]
First Deployment
**Works Great**
DEV--------------------TEST(existing tables)
Table1----------------- Table1
Table2----------------- Table2
Table3----------------- NOT EXIST

Second Deployment 
**Issue Occurs** ---------- _"The table object name Table3 does not exist"_
DEV--------------------TEST(existing tables)
Table1-----------------Table1
Table2-----------------Table2
Table3_1---------------Table3-------------------- _renaming Table3 to Table3_1_
